fastlane/Matchfile

```diff
@@ -1,7 +1,4 @@
-git_url "git@github.com:mapbox/apple-certificates.git"
-type "appstore" # The default type, can be: appstore, adhoc, or development
-app_identifier "com.mapbox.examples"
-
-# For consistency, only one username should ever be used to update certs/profiles.
-# See 1Password for account credentials.
-username "applemachine@mapbox.com"
\ No newline at end of file
+git_url("git@github.com:mapbox/apple-certificates.git")
```

Comment:
This is mostly a duplicate of `/Apps/.fastlane/Matchfile`

Reply:
Yeah, deleted this folder